src/apis/experimental/enum_defaults.rs

```diff
@@ -0,0 +1,56 @@
+use super::httproutes::{
```

Comment:
So your new enum generator code manages this file now, but we don't have a comment to indicate this to code readers (e.g. like other files in this directory, kopium will leave a helpful comment at the top indicating these are generated files, so people can find the generator and/or don't feel inclined to change the file directly).

We should make sure our new generator adds a "generated by" comment with some breadcrumbs to tell them where the generator lives.